manage

```diff
@@ -550,7 +550,7 @@ getAgentVersion(){
 
 createNetwork() {
   if [[ -z `docker network ls -q --filter "name=aath_network"` ]]; then
-    docker network create aath_network > /dev/null
+    docker network create aath_network --subnet=172.28.0.0/24 > /dev/null
```

Comment:
@WadeBarnes — is hard-coding this value in this context safe on different docker platforms (especially GHA?). I’m not up on the latest best practices for this.

For background: The goal is that we can have a redis-cluster `service` started up before running tests, and if available one or more of the test-agents can use the redis cluster as they see fit. In our case, we have a variation of the ACA-Py test agent that uses redid.

Reply:
That should work.  It's defining the subnet (address ranges) for the specific docker network.  It shouldn't conflict with anything in GHA as they typically use the Case A subnet ranges (10.0. 0.0 to 10.255. 255.255).